ops/docker-compose-dev.yml

```diff
@@ -1,7 +1,8 @@
 version: '2'
 services:
-  db:
-    build: docker/images/mysql
+  mysql:
+    container_name: mysql
```

Comment:
its nice to see short container names like `mysql` or `php-cli` instead of `ops_php-cli` and `ops_mysql` when you do `docker ps`